packages/nc-gui/helpers/tiptap/extensions/tiptap/tight-lists.ts

```
@@ -0,0 +1,53 @@
+import { Extension } from '@tiptap/core'
+
+export interface MarkdownTightListsOptions {
+  tight: boolean
+  tightClass: string
+  listTypes: Array<'bulletList' | 'orderedList'>
+}
+
+export interface MarkdownTightListsOptionsStorage {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)